packages/sdk-react/src/utils/compile-lingui-dynamic-messages.test.ts

```diff
@@ -18,12 +18,28 @@ describe('compileLinguiDynamicMessages()', () => {
     );
   });
 
-  it('uses context for key hashing', async () => {
+  it('uses multiple context for key hashing', async () => {
     expect(
       await compileLinguiDynamicMessages({
-        'Hey Monkeys!': { message: 'Hey Jirafs!', context: 'animals' },
+        'Hey Monkeys!': [
+          { msgstr: 'Hey Jirafs!', msgctxt: 'animals' },
+          { msgstr: 'Hey Elephant!', msgctxt: 'giant-animals' },
+        ],
       })
-    ).toEqual({ sVfpdY: 'Hey Jirafs!', 'Hey Monkeys!': 'Hey Jirafs!' });
+    ).toEqual({
+      sVfpdY: 'Hey Jirafs!',
```

Comment:
Can you please explain where you get these keys?

Reply:
If I understand correctly, it's a hash of `Hey Jirafs!`

Reply:
@hypsteria, thank you, clarified implementation using `generateLinguiMessageId(...)` as a key: `[generateLinguiMessageId('Hey Monkeys!', 'animals')]: ...`

Reply:
got it, thanks!